### PR TITLE
feat(client signin): implement signin at client-side

### PR DESCRIPTION
**Pull request by Oloyedesinmiloluwa**
#### What does this PR do?
have client side login implementation
#### Description of Task to be completed?
Enable client-side sign in endpoint
#### How should this be manually tested?
- After repo clone run npm install and npm run start:dev
- You can test the signin route afterward
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/157931582

### CODE REVIEW

File: client/signin.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  fetch('https://m-tracker.herokuapp.com/api/v1/auth/login', {
+    method: 'POST',
+    headers: { 'content-type': 'application/json' },
+    body: JSON.stringify(inputData)
+  })
+    .then((response) => { return response.json(); })
+    .then((data) => {
+      if (data.message === 'Login successful') {
+        window.location.href = 'index.html';
+        sessionStorage.setItem('token', data.token);
```
Comment (benfluleck) [inline client/signin.js]: why are you using sessionStorage being used here?

File: server/src/model/seed.js [hunk 2]
```diff
@@ -4,20 +4,22 @@ import { connectionString } from '../config/config';
...
 
-const seedUserData = { text: 'INSERT INTO users (firstname, lastname, email, password, role) VALUES($1, $2,$3,$4,$5)',
-  values: ['Sinmi', 'John', 'sinmiloluwasunday@yahoo.com', hashedPassword, 'admin'] };
+const seedUserData = {
+  text: 'INSERT INTO users (firstname, lastname, email, password, role) VALUES($1, $2,$3,$4,$5)',
+  values: ['Sinmi', 'John', 'sinmiloluwasunday@yahoo.com', hashedPassword, 'admin']
+};
 const seedRequestData1 = `INSERT INTO requests (title, description, category, image, status, dated, userId) VALUES
-('Faulty Fan', 'we have a fault', 'electrical', 'IMAGE', 'approved', '2018-12-13', 1);`;
+('Damaged Chair', 'we have a fault', 'electrical', 'chair.jpg', 'approved', '2018-12-13', 1);`;
```
Comment (benfluleck) [inline server/src/model/seed.js:12]: use cloudinary links to render these images.

File: client/signin.js (hunk 1 above)
Comment (Oloyedesinmiloluwa) [reply client/signin.js]: No special reason, do you suggest I use another ? will localStorage do ?

Comment (benfluleck) [reply client/signin.js]: Go and research the difference between localStorage and sessionStorage there is a use case for both

File: server/src/model/seed.js (hunk 2 above)
Comment (Oloyedesinmiloluwa) [reply server/src/model/seed.js:12]: alright I will, thanks

File: client/signin.js (hunk 1 above)
Comment (Oloyedesinmiloluwa) [reply client/signin.js]: alright

**Pull request merged**